src/components/select/select.js

```diff
@@ -1582,4 +1582,14 @@ function SelectProvider($$interimElementProvider) {
     }
     return isScrollable;
   }
+
+}
+
+function shouldHandleKey(ev) {
+  var char = String.fromCharCode(ev.keyCode);
+  var isNonUsefulKey = (ev.keyCode <= 31);
+  var isLeftRightSelectKey = (ev.keyCode >= 91 && ev.keyCode <= 93);
+  var isFnLockKey = (ev.keyCode >= 112 && ev.keyCode <= 145);
+
```

Comment:
I think it's safe to abort if the user press `Ctrl` or `Meta` or `Alt`.  I say this because `master` locked me out from refreshing the page with Ctrl+Shift+R.

`Shift` alone should be okay. I tried with with `AltGr` and special Latin characters (ñÑéóö) and `Alt` is true while holding `AltGr` but gets set to `false` when I press AltGr+N. `ñ` is detected as it's own key with `Alt` set to `false`, so it looks safe.